app/components/login-component/index.scss

```diff
@@ -0,0 +1,118 @@
+.container {
+  height: 100vh;
+}
+
+.inner-container {
+  width: 100%;
+  display: flex;
+  align-items: center;
+  flex-direction: column;
+  padding-top: 3em;
+}
+
+.login-container {
+  padding: 0em;
+  background-color: var(--white);
+  border-radius: 5px;
+  min-width: 26em;
+  box-shadow: 2px 2px 17px -1px rgba(0, 0, 0, 0.5);
+  margin-bottom: 3em;
+  margin-top: 3em;
+}
+
+.login-title {
+  font-size: 1.72rem;
+  text-align: center;
+  margin-top: 2em;
+}
+
+.login-footer {
+  margin-top: 3em;
+  border-top: 1px solid #efefef;
```

Comment:
shall we use style var here?

Reply:
ya wanted to but...it is not defined yet...these were the original colors from previous components...once it is defined will migrate.